admin/settings-options.php

```diff
@@ -366,6 +370,23 @@ function wpuf_settings_fields() {
                 'options' => wpuf_get_gateways()
             )
         ) ),
+        'wpuf_guest_mails' => array(
+            array(
+                'name'       => 'guest_email_subject',
+                'label'    => __( 'Guest Email Subject', 'wpuf' ),
+                'desc'     => __( 'This sets the subject of the emails sent to guest users', 'wpuf' ),
+                'default'  => 'Please Confirm Your Email to Get the Post Published!',
+                'type'     => 'text',
+            ),
+            array(
+                'name'       => 'guest_email_body',
+                'label'    => __( 'Guest Email Body', 'wpuf' ),
+                'desc'     => __( "This sets the body of the emails sent to guest users. Please DON'T edit the '{click}' part", 'wpuf' ),
+                'default'  => "Hey There, \r\n\r\nWe just received your guest post and now we want you to confirm your email so that we can verify the content and move on to the publishing process.
```

Comment:
Update the body like this:

```
Hey There,

We just received your guest post and now we want you to confirm your email so that we can verify the content and move on to the publishing process.

Please click the link below to verify:

{activation_link}

Regards,
%sitename%```